isb_lib/core.py

```diff
@@ -65,7 +63,15 @@ def _shouldAddMetadataValueToSolrDoc(metadata: typing.Dict, key: typing.AnyStr)
     shouldAdd = False
     if key in metadata:
         value = metadata[key]
-        if type(value) is list:
+        if key =='latitude':
+            shouldAdd = -90.0 <= value <= 90.0
+            if not shouldAdd:
+                getLogger().error("Invalid latitude %f", value)
+        elif key == 'longitude':
```

Comment:
Just a note that longitude is an odd one to check for range since it's cyclical. Technically any value can work since it's multiples of 360. -180 ... 180 is a general convention though, and this works.